frontend/src/lib/services/RecaptchaService.ts

```diff
@@ -47,3 +48,15 @@ export function recaptchaServiceFactory(
 ): RecaptchaService {
 	return mode === 'test' ? new MockRecaptchaService() : new RecaptchaServiceImpl(props);
 }
+
+export async function loadRecaptchaProps(mode: Mode): Promise<RecaptchaServiceProps> {
+	let recaptchaKey: string;
+	if (mode == 'production') {
+		recaptchaKey = await axios.get('/recaptcha-key');
+	} else if (mode == 'development') {
+		recaptchaKey = String(import.meta.env.VITE_GOOGLE_RECAPTCHA_KEY);
+	} else {
+		recaptchaKey = 'sdfafdaf';
+	}
```

Comment:
I think it's okay to default to `VITE_GOOGLE_RECAPTCHA_KEY` if it is a non-empty string and then try `/recaptcha-key` if it is not.

Reply:
Updated here => https://github.com/UnityFoundation-io/Libre311/pull/119/commits/4fbccd4b8ecf37ecae26d94e291a56acfb8dce6a